File: Modul 4/Bahasa Python (modul 4)/PRAK402-2310817320015-AlfisyahRinaFajriati.py
print("Input")
bil = int(input(""))

print("\nOutput")
for i in range(1, bil + 1, 2):
    print(i, end=" ")

print()
for i in range(bil, 1, -1):
    if i % 2 == 0:
      print(i, end=" ")

File: Modul 4/Bahasa C (modul 4)/PRAK402_2310817320015_AlfisyahRinaFajriati.c
#include <stdio.h>
int main(){
    int batas;
    printf("Input\n");
    scanf("%d", &batas);

    printf("\nOutput\n");
    for(int i = 1; i <= batas; i++) {
        if(i % 2 != 0)
            printf("%d ", i);
    }
    printf("\n");
    for(int i = batas; i >= 1; i--) {
        if(i % 2 == 0)
            printf("%d ", i);
    }
    return 0;
}